slashing-protection/src/test/java/tech/pegasys/web3signer/slashingprotection/DbSlashingProtectionTest.java

```diff
@@ -238,6 +254,50 @@ public void cannotSignAttestationWhichWasPreviouslySignedButBelowTargetWatermark
     verify(lowWatermarkDao).findLowWatermarkForValidator(any(), eq(VALIDATOR_ID));
   }
 
+  @Test
+  public void cannotSignAttestationWhenSourceIsAtOrBeyondHighWatermark() {
+    final SignedAttestation attestation =
```

Comment:
I wanted to make the source and target the same value to make sure the source is actually being compared against the high watermark without making the source > target since I don't think that can happen in reality.